### DISCUSSION TITLE
No resources were found in Search UI after creating the SearchConfig resources in the project

### DISCUSSION THREAD

**Discussion by QXgu** [BlueBrain/nexus#2796] [Q&A] [answered]
Hi,
I currently met the problem that no resources were found in Search UI after creating the SearchConfig resources in the project. I have run the query in the view editor and it can return resources but the Search Page still not working. I found out that after I setting up my SearchConfig resources as shown below, the fields of the table in the Search still remain the same as the default. Is there anything wrong with the content of the SearchConfig resource? Is there any way to solve this problem?
`{ "@id": "SearchConfig", "@type": [ "https://bluebrain.github.io/nexus/vocabulary/SearchConfig" ], "fields": [ { "dataIndex": "@type", "displayIndex": 0, "key": "@type", "sortable": true, "title": "Type1" } ], "view": "https://my-nexus-deployment.com/v1/views/VRE_testing/VREDatasets/https%3A%2F%2Fbluebrain.github.io%2Fnexus%2Fvocabulary%2FdefaultElasticSearchIndex", "label": "SearchConfig", "description": "global dataset search" }`

Best,
Amy

**Comment by dhaneshnm** [comment, marked as answer]
Some of the concerns are addressed here: https://github.com/BlueBrain/nexus/discussions/2763. 
It could also be due to a delay in indexing of the resource.
